src/pages/[postcode]/material/RecycleAtHome.tsx

```diff
@@ -0,0 +1,161 @@
+
+function ManySchemes({
+  schemes,
+  schemesWithContainers,
+}: {
+  readonly schemes: DryScheme[];
+  readonly schemesWithContainers: DryScheme[];
+}) {
+  const { postcode } = useParams();
+  const tContext = 'material.recycleAtHome.manySchemes';
+  const allSchemesRecycle = schemesWithContainers.length === schemes.length;
+  const sortedSchemes = schemes.toSorted((scheme) => {
+    return scheme.containers.length > 0 ? -1 : 1;
+  });
+
+  return (
+    <>
+      <p className="text-size-sm">
+        <Trans
+          i18nKey={`${tContext}.collection${allSchemesRecycle ? 'All' : 'Some'}`}
+          components={{ bold: <strong /> }}
+        />
+      </p>
+      <ul role="list" className="list-style-none diamond-spacing-bottom-md">
+        {sortedSchemes.map((scheme) => {
+          const safeSchemeName = encodeURIComponent(scheme.name);
+          const recyclable = scheme.containers.length > 0;
+
+          return (
+            <li key={scheme.name} className="diamond-spacing-bottom-sm">
+              <locator-icon-link>
+                <Link
+                  to={`/${postcode}/home/collection?scheme=${safeSchemeName}`}
+                >
+                  <locator-icon-circle>
+                    <locator-icon
+                      icon={recyclable ? 'tick' : 'close'}
+                      color={recyclable ? 'positive' : 'negative'}
+                    ></locator-icon>
+                  </locator-icon-circle>
+                  {scheme.name}
+                </Link>
+              </locator-icon-link>
+            </li>
+          );
+        })}
+      </ul>
+    </>
+  );
+}
+
+function OneScheme({ scheme }: { readonly scheme: DryScheme }) {
+  const { t } = useTranslation();
+  const tContext = 'material.recycleAtHome.oneScheme';
+  const firstTwoContainers = scheme.containers.slice(0, 2);
+  const remainingContainers = scheme.containers.slice(2);
+
+  return (
+    <>
+      <p className="text-size-sm">{t(`${tContext}.collection`)}</p>
+      <ul role="list" className="list-style-none diamond-spacing-bottom-md">
+        {firstTwoContainers.map((container) => (
+          <li key={container.name} className="diamond-spacing-bottom-sm">
+            <locator-container>
+              <locator-container-svg
+                name={container.name}
+                body-colour={container.bodyColour}
+                lid-colour={container.lidColour}
+              />
+              {containerName(container)}
+            </locator-container>
+          </li>
+        ))}
+        {remainingContainers.length > 0 && (
+          <li>
+            {t(`${tContext}.otherContainers`, {
+              count: remainingContainers.length,
+            })}
+          </li>
+        )}
+      </ul>
+    </>
+  );
+}
+
+export default function RecycleAtHome({
+  schemes,
+}: {
+  readonly schemes: DryScheme[];
+}) {
+  const { postcode } = useParams();
+  const { t } = useTranslation();
+  const schemesWithContainers = schemes.filter(
```

Comment:
Could we name this something like `collectingSchemes`, to make it more obvious what this is? I initially thought that we were somehow getting schemes returned by the API that had no containers.